core/indexer/elasticsearch.go

```diff
@@ -223,6 +237,72 @@ func (ei *elasticIndexer) SaveBlock(
 	}
 }
 
+// SaveRoundInfo will save on elastic search information about round
```

Comment:
_SaveRoundInfo will save data about a round on elastic search_ sounds better

Reply:
not done yet

Reply:
Done.